app/controllers/api/v1/items_controller.rb

```diff
@@ -83,6 +84,10 @@ def update
       end
     end
 
+    def browse
+      render json: @items.accepted, each_serializer: serializer
+    end
+
```

Comment:
@swatijadhav cc @mcm-ham I think it might be good to have a separate ```BrowseController``` here to separate out the logic and future work. It should still return ```@items.accepted``` but I like the idea of separating the controller actions for the new app. That way we can be relatively sure that change in the browse.goodcity spec will have less of an impact on app.goodcity etc etc